src/einsteinpy/utils/__init__.py

```diff
@@ -1 +1,3 @@
 from .scalar_factor import scalar_factor, scalar_factor_derivative
+
+__all__ = ["scalar_factor", "scalar_factor_derivative"]
```

Comment:
I am unsure, if we want these to be exposed at the moment. @shreyasbapat Thoughts?

Reply:
I'll wait for @shreyasbapat 's reply on this

Reply:
@Qbiwan Let's keep these hidden for now, as some work is yet to be done towards FLRW spacetime.